hatchet_sdk/clients/admin.py

```diff
@@ -41,33 +41,33 @@ def new_admin(config: ClientConfig):
     return AdminClient(config)
 
 
-class ScheduleTriggerWorkflowOptions(TypedDict):
+class ScheduleTriggerWorkflowOptions(TypedDict, total=False):
```

Comment:
adding `total` here allows you to create a dictionary that does not include all of these keys and have mypy still allow it to consider it as conforming to this type. we should use Pydantic going forward, but this is fine for now IMO